idea_town/frontend/static-src/app/lib/page-manager.js

```diff
@@ -27,7 +27,13 @@ export default class PageManager {
       'error': ErrorPage
     };
 
-    this._viewSwitcher = new AmpersandViewSwitcher(opts.pageContainer);
+    this._viewSwitcher = new AmpersandViewSwitcher(opts.pageContainer, {
+      show: (view) => {
+        document.title = 'Idea Town - ' + view.pageTitle || 'Idea Town';
```

Comment:
Whenever we get into localization, this could be a gotcha spot. Simple concatenations don't generally work across languages. Instead you might want to make this a template string that gets filled with `view.pageTitle` - e.g. something like <code>\`Idea Town - ${view.pageTitle}\`</code> and we can expose that template string itself for localization.

Of course, since we haven't even started on l10n, this is probably putting the cart before the horse :)